src/allmydata/frontends/magic_folder.py

```diff
@@ -632,6 +675,15 @@ def is_pending(self, relpath_u):
         return relpath_u in self._pending
 
     def _notify(self, opaque, path, events_mask):
+        # Twisted doesn't seem to do anything if our callback throws
+        # an error, so...
+        try:
+            return self._real_notify(opaque, path, events_mask)
+        except Exception as e:
+            self._log(u"error calling _real_notify: {}".format(e))
+            twlog.err(Failure())
```

Comment:
Where does `self._log` go?  Not the Twisted log, I suppose.  The flog?

Not sure if this behavior is relevant or not.

```
>>> from twisted.python.log import err
>>> from twisted.python.failure import Failure
>>> err(Failure(ValueError("foo")))
Unhandled Error
Traceback (most recent call last):
Failure: exceptions.ValueError: foo

>>> err(Failure(ValueError("foo")), "Some context")
Some context
Traceback (most recent call last):
Failure: exceptions.ValueError: foo
```

Maybe it just suggests there should be a 2nd argument to the `twlog.err` call.

Reply:
it does end up in the flog, via client for some reason?